### ISSUE TITLE
unity-tweak-tools error

### ISSUE THREAD

**Issue by xSQL** [shiznix/unity-gentoo#125]
When i'm trying emerge unity-tweak-tools, i get this error:
ERROR: module apport not found
ImportError: No module named 'values'

**Comment by susman** [comment]
I think the problem display there is a bit further in the crash log:
```
PermissionError: [Errno 13] Permission denied: '/var/tmp/portage/unity-extra/unity-tweak-tool-0.0.6_p_p03/homedir/.config/unity-tweak-tool'
```
which is true:
```
→ls -la /var/tmp/portage/unity-extra/unity-tweak-tool-0.0.6_p_p03/homedir/
total 4
drwxr-xr-x 6 portage portage   62 Dec 16 12:08 .
drwxr-xr-x 7 portage portage 4096 Dec 16 12:08 ..
drwxr-xr-x 2 root    root       6 Dec 16 12:08 .cache
drwxr-xr-x 2 root    root       6 Dec 16 12:08 .config
drwxr-xr-x 3 root    root      18 Dec 16 12:08 .local
drwxr-xr-x 2 portage portage   29 Dec 16 12:08 python3.4
```

**Comment by xSQL** [comment]
Yes, but what this means? What i need to do?

**Comment by susman** [comment]
You can wait for fix, it won't take long, or you can dirty-hack it by running while true chmod during emerge :D

**Comment by xSQL** [comment]
Thanks a lot.

**Comment by shiznix** [comment]
Hi I too am experiencing this build error, however I did have it successfully build and install at some point as I had this version installed (using python3.4) prior to looking at this bug report.

Could be a bug in unity-tweak-tools or python-distutils-extra, haven't tracked down which yet...

**Comment by shiznix** [comment]
Nope totally my bad was using gnome2_environment_reset in the ebuild when I shouldn't have.

Should be fixed in commit 5030d81dc86c795dc214c357c59befe08510fcbd
